packages/web-microkernel/src/RealmConnection.js

```diff
@@ -0,0 +1,400 @@
+import * as _ from './lodash-es/lodash.js'
```

Comment:
would be nice to tree-shake other lodash functions before we release

Reply:
We're not going to use Lodash—it's temporary filler.